WTWSMS/localisation/AO_misc.csv

```diff
@@ -6,3 +6,5 @@ COMPL_WORTHY;worthy;digne;;;;;;;;;;;;x
 INSULT_NAIVE;naive;na�f;;;;;;;;;;;;x
 INSULT_IGNORANT;ignorant;ignorant;;;;;;;;;;;;x
 INSULT_HARSH;harsh;dur;;;;;;;;;;;;x
+GOD_PROMETHEUS;Prometheus;;;;;;;;;;;;;x
+GOD_KRONOS;Kronos;;;;;;;;;;;;;x
```

Comment:
Why did you add this here?

Reply:
these keys were reported missing by the Validator

Reply:
Okay.